Libs/MRML/Logic/vtkMRMLApplicationLogic.h

```diff
@@ -289,6 +291,15 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLApplicationLogic
   /// Font file path is set to the one specified in FontFileName property in this object.
   void UseCustomFontFile(vtkTextProperty* textProperty);
 
+  // Translation function for logic classes
```

Comment:
```suggestion
  /// Translation function for logic classes
```

Reply:
I suggest you review the other comments so that they are consistently specified.